src/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/common/DirectoryRequest.cs

```diff
@@ -2,229 +2,120 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.ComponentModel;
+using System.Collections.Specialized;
+
 namespace System.DirectoryServices.Protocols
 {
-    using System;
-    using System.Collections;
-    using System.IO;
-    using System.Diagnostics;
-    using System.Globalization;
-    using System.ComponentModel;
-    using System.Collections.Specialized;
-    using System.Security.Permissions;
-
     public abstract class DirectoryRequest : DirectoryOperation
     {
-        internal DirectoryControlCollection directoryControlCollection = null;
+        internal DirectoryControlCollection directoryControlCollection;
```

Comment:
Why is this field still here? Previously it was being initialized by DirectoryRequest's ctor and returned from Controls, now it appears to be unuused.